src/main/resources/public/jsx/DatasetAutosuggest.jsx

```diff
@@ -62,7 +62,7 @@ export default class DatasetAutosuggest extends React.Component {
         this.setState({ suggestions: [] })
     }
     onSuggestionSelected = (event, { suggestion }) => {
-        this.props.onSelect(suggestion)
+        this.props.onSelect("name", suggestion)
```

Comment:
I did this to avoid redundancy of functions in DatasetConfigurer "handleDelete", "handleAddition"